Services/DatabaseService.cs

```diff
@@ -41,15 +41,39 @@ public async Task SetPrefixAsync(ulong guildId, string prefix)
                 settings = new GuildSettings { GuildId = guildId, Prefix = prefix };
                 dbContext.GuildSettings.Add(settings);
             }
-            else
+            
+            settings.Prefix = prefix;
+            await dbContext.SaveChangesAsync();
+        }
+    }
+
+    public async Task<ulong?> GetMutedRoleAsync(ulong guildId)
+    {
+        using (var dbContext = new BotDbContext())
+        {
+            var settings = await dbContext.GuildSettings.FindAsync(guildId);
+            return settings?.MutedRoleId;
+        }
+    }
+
+    public async Task SetMutedRoleAsync(ulong guildId, ulong? mutedRoleId)
+    {
+        using (var dbContext = new BotDbContext())
+        {
+            var settings = await dbContext.GuildSettings.FindAsync(guildId);
+
+            if (settings == null)
             {
-                settings.Prefix = prefix;
+                settings = new GuildSettings { GuildId = guildId };
+                dbContext.GuildSettings.Add(settings);
             }
```

Comment:
Put that into a function, to avoid code duplication. I figure you're gonna have to use (or already do) something like this a lot of times in a lot of places, basically for every config change.

Reply:
is a function for 5 lines of code necessary?

Reply:
who cares